model/work.py

```diff
@@ -222,6 +230,13 @@ class Work(Base):
     # catalog.
     marc_record = Column(String, default=None)
 
+    # These fields are potentially large and can be deferred if you
+    # don't need all the data in a Work.
+    LARGE_FIELDS = [
+        'simple_opds_entry', 'verbose_opds_entry', 'marc_record',
+        'summary_text',
+    ]
+
```

Comment:
This should speed up the WorkReaper quite a bit.

Reply:
So the `defer` for these values tells the query to not load these columns unless if they are directly accessed, right?

Reply:
Right, if that happened they would be fetched in a separate query.